### PR TITLE
feat: axios instance made with new configuration

### PR DESCRIPTION
**Pull request by MohannadSabaa**
I have created two .env files (.env.development, .en.production), added axiosInstance with configuration, and fixed a conflict in therapist module..

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,13 +6,15 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint src --ext ts,tsx",
+    "lint:fix": "eslint src --ext ts,tsx --fix",
```
Comment (LinaYahya) [inline package.json]: please revert this change

**Review by ahmedmurtaja — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by raghdarizq — APPROVED** [review]

**Review by YasserAmro0 — APPROVED** [review]
